usama/sprint1/cdk.out/asset.0e9db55350e39158be9550d074e22f47df7cc37733a302a79954c0f2729738d0/web_health_publisher.py

```diff
@@ -0,0 +1,45 @@
+import datetime
+import urllib3
+import urllib.request
+import json
+import constants
+
+
+def health_monitor(event, context):
+    
+    url = constants.URLS_TO_MONITOR
+    
+    availability = get_availability(url)
+    
+    latency = get_latency(url)
+    
+    result = {'Availability': availability, 'Latency': "{}ms".format(latency)}
+    
+    return result
+
+
+
+def get_availability(web_url):
+
+    res = urllib.request.urlopen(urllib.request.Request(
```

Comment:
- You can create a single function to measure and latency to save the memory and computational power required.
- By doing this, you don't need to get HTTP requests twice

Reply:
Here, I would like to say that to be our modularized each function should do only one thing. I guess it was discussed in class.